.gitignore

```diff
@@ -65,4 +65,5 @@ package-lock.json
 yarn.lock
 
 # ignore sqlite3 database
-*.db3
\ No newline at end of file
+*.db3
+index.js
```

Comment:
You're having `index.js` in `.gitignore`,  can you explain why?

Reply:
Yea, It was because I used a solution to get around the jest problem with 5000 in use. Shaun told me to remove the index.js file until we spoke in standup about solution (I think Wasiu has fixed in his PR)